src/eva/vision/utils/convert.py

```diff
@@ -6,19 +6,16 @@
 import numpy.typing as npt
 
 
-def to_8bit(image_array: npt.NDArray[Any]) -> npt.NDArray[np.uint8]:
-    """Casts an image of higher bit image (i.e. 16bit) to 8bit.
+def to_8bit(image: npt.NDArray[Any]) -> npt.NDArray[np.uint8]:
+    """Casts an image array of higher bit image (i.e. 16bit) to 8bit.
 
     Args:
-        image_array: The image array to convert.
+        image: The image array to convert.
 
     Returns:
-        The image as normalized as a 8-bit format.
+        The image as uint8 (0, 255) array.
     """
-    if np.issubdtype(image_array.dtype, np.integer):
-        image_array = image_array.astype(np.float64)
-
-    image_scaled_array = image_array - image_array.min()
-    image_scaled_array /= image_scaled_array.max()
-    image_scaled_array *= 255
-    return image_scaled_array.astype(np.uint8)
+    image_scaled = image - image.min()
+    image_scaled /= image_scaled.max()
```

Comment:
Shouldn't this be `image_scaled /= (image_scaled.max() - image_scaled.min())`?

Reply:
why is that? because it was already shifted from the substraction of the image min

Reply:
here it uses less extra variables as first the minimum value is moved to zero, thereafter one can take the maximum value to get the full range (max-min) of the specific image.

the following are the same:
```
data_min = np.min(data)
data_max = np.max(data)
scaled_data = (data - data_min) / (data_max - data_min)
```

and
```
data -= data.min()
data /= data.max()
```

here: https://stackoverflow.com/a/70783585

Reply:
Ah got you, makes sense!